doc/tutorials/02-charged_system/02-charged_system-1.ipynb

```diff
@@ -380,68 +680,129 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Plot the distribution functions\n",
-    "plt.figure(figsize=(10, 6), dpi=80)\n",
-    "plt.plot(r[:], rdf_00[:], label='Na$-$Na')\n",
-    "plt.plot(r[:], rdf_01[:], label='Na$-$Cl')\n",
-    "plt.xlabel('$r$', fontsize=20)\n",
-    "plt.ylabel('$g(r)$', fontsize=20)\n",
-    "plt.legend(fontsize=20)\n",
-    "plt.show()"
+    "def add_salt(system, anion_params, cation_params):        \n",
+    "    for _ in range(anion_params['number']):\n",
```

Comment:
Ideally, create the particles in a single call and keep a reference to the resulting slice.